proto/pbsubscribe/subscribe.pb.go

```diff
@@ -320,6 +320,7 @@ type SubscribeRequest struct {
 	// receive events (e.g. health events for a particular service).
 	//
 	// Types that are assignable to Subject:
+	//
```

Comment:
Did you have to edit this, or did you run `make proto` and it did the right thing?

Reply:
`make proto` formatted everything